### PR TITLE
clientcontainer beanvalidation fat

### PR DESCRIPTION
**Pull request by awisniew90**
Port clientcontainer beanvalidation FAT and run with bval v1.1 and bval v2.0

### CODE REVIEW

File: dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/apps
```
Comment (nmittles) [inline dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.gitignore]: We shouldn't be checking in .gitignore files.

File: dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.classpath [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nmittles) [inline dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.classpath]: Remove the "<classpathentry combineaccessrules..." lines, as the bnd.bnd file handles this.  Update to include:
<classpathentry kind="con" path="aQute.bnd.classpath.container"/>

File: dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.classpath [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/ApacheBvalConfig.jar/src"/>
+	<classpathentry kind="src" path="test-applications/beanvalidation.jar/src"/>
+	<classpathentry kind="src" path="test-applications/BeanValidationCDI.jar/src"/>
+	<classpathentry kind="src" path="test-applications/defaultbeanvalidation.jar/src"/>
+	<classpathentry kind="src" path="test-applications/DefaultBeanValidationCDI.jar/src"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/fattest.simplicity"/>
+	<classpathentry kind="lib" path="/ant_build/lib/junit.jar"/>
```
Comment (nmittles) [inline dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.classpath]: I believe this junit.jar entry should be removed as well.  It should be part of the bnd bundle path as "/build.sharedResources/lib/junit/old/junit.jar"

Can we change all binary checkins of junit.jar to point to /build.sharedResources/lib/junit/old/junit.jar instead?
For example com.ibm.ws.clientcontainer.beanvalidation.fat.ApacheBvalConfig_11/lib/global/junit.jar

File: dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.classpath (hunk 2 above)
Comment (awisniew90) [reply dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.classpath]: Done

**Comment by awisniew90** [comment]
#build

**Comment by awisniew90** [comment]
#build

**Pull request merged**